src/fsharp/FSharp.Build/Microsoft.FSharp.NetSdk.props

```diff
@@ -30,9 +30,33 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
   </Choose>
 
   <PropertyGroup>
-    <WarningLevel Condition=" '$(WarningLevel)' == '' ">3</WarningLevel>
     <EnableDefaultCompileItems Condition=" '$(EnableDefaultCompileItems)' == '' ">false</EnableDefaultCompileItems>                                <!--- Do not glob F# source files -->
+    <EnableDefaultNoneItems Condition=" '$(EnableDefaultNoneItems)' == '' ">false</EnableDefaultNoneItems>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <WarningLevel Condition=" '$(WarningLevel)' == '' ">3</WarningLevel>
     <DefaultProjectTypeGuid Condition=" '$(DefaultProjectTypeGuid)' == '' ">{F2A71F9B-5D33-465A-A702-920D77279786}</DefaultProjectTypeGuid>         <!-- F# project type -->
+    <PlatformTarget>AnyCPU</PlatformTarget>
```

Comment:
Good point on the platform target.  I think Tailcalls false is a bad choice for debug.  Similarly I think prefer32bit=true is a poor default.  We can always measure things at a later date and modify the defaults if we need.